packages/esbuild-plugin-lit-css/README.md

```diff
@@ -96,14 +96,14 @@ class CSSinCSS extends FASTElement {}
 ### Usage with Sass, Less, PostCSS, etc.
 
 To load scss files:
-1. Specify the [`js` loader in your esbuild config](https://esbuild.github.io/api/#loader) for `.scss` files,
-2. Specify the `filter` option to `litCssPlugin` to include scss files
-2. Define a `transform` function in the plugin options.
+1\. Specify the [`js` loader in your esbuild config](https://esbuild.github.io/api/#loader) for `.scss` files,
+2\. Specify the `filter` option to `litCssPlugin` to include scss files
+2\. Define a `transform` function in the plugin options.
```

Comment:
3? :)

Reply:
yes, and markdown automatically fixes that for you, so whatev